src/applications/vaos/new-appointment/sass/_calendar.scss

```diff
@@ -74,7 +74,7 @@
   background-color: $color-gray-lightest;
   border: 1px solid $color-gray-medium;
   border-radius: 5px;
-  padding: 12px;
+  padding: 5px;
```

Comment:
It looks like this flattens the buttons in a way we don't want:

![Screen Shot 2021-01-04 at 10 19 15 AM](https://user-images.githubusercontent.com/634932/103550069-57bc5580-4e76-11eb-927c-5cc8a857ff9c.png)

Reply:
Fixed, that was an accidental change on my part.